kvmagent/kvmagent/plugins/zbs_storage_plugin.py

```
@@ -262,6 +264,53 @@ def luks_clone(self, req):
+                raise Exception('installPath and targetInstallPath are required')
+            encrypted_dek = cmd_attr(cmd, 'encryptedDek')
+            if not encrypted_dek:
+                raise Exception('encryptedDek is required')
+            virtual_size = int(cmd_attr(cmd, 'virtualSize') or 0)
+            if virtual_size <= 0:
+                raise Exception('virtualSize is required and must be greater than 0')
+            target_encrypted = cmd_attr(cmd, 'targetEncrypted')
+            source_is_luks = self._is_luks_volume(src_path)
+            if source_is_luks == bool(target_encrypted):
+                raise Exception('ZBS source format does not match requested encryption conversion')
+            secret_file_provider = lambda: volume_secret.luks_secret_channel(encrypted_dek)
```

Ruff (0.15.21)

[error] Do not assign a `lambda` expression, use a `def`

Rewrite `secret_file_provider` as a `def`

(E731)